### ISSUE TITLE
Issue with exemple

### ISSUE THREAD

**Issue by Ventricule** [bewithjonam/mapboxgl-spiderifier#8]
Hello,
I would like to use spiderifier to show markers at the exact same point on a map.
I'm starting from your cluster spiderifier exemple but I'm facing an error in this part of code :
```
getClusterChildren(features[0].properties.cluster_id, Math.floor(map.getZoom()), true, function(data){
    var markers = _.map(data.children, function(child){
        return child.properties;
    });
    spiderifier.spiderfy(features[0].geometry.coordinates, markers);
})
```
I'm facing this error : `TypeError: e is not a function   actor.js:83:12`
I'm wondering why the function getClusterChildren() have 4 parameters while the official doc [is giving 2 parameters](https://www.mapbox.com/mapbox-gl-js/api/#geojsonsource#getclusterchildren) ?

Thank you !
Pierre

**Comment by bewithjonam** [comment]
@Ventricule 

Previously getClusterChildren and getClusterLeaves methods were not exposed by mapbox-gl. (Mapbox internally uses supercluster library for clustering which had those methods) 

I had created a fork and exposed those methods with different method signature/callback. And used it for my projects and created the example page.

Now, mapboxgl-js has exposed methods getClusterLeaves/getClusterChildren in version v0.47.0. 

I have updated the [readme-getting features in a cluster from mapbox-gl and spiderfying them](https://github.com/bewithjonam/mapboxgl-spiderifier#getting-features-in-a-cluster-from-mapboxgl-and-spiderfying-them) and the [example](https://bewithjonam.github.io/mapboxgl-spiderifier/docs/example-mapbox-gl-cluster-spiderfy.html).
